src/test/config.ts

```diff
@@ -32,7 +32,8 @@ export const config: FlowEditorConfig = {
     activity: '',
     simulateStart: '',
     simulateResume: ''
-  }
+  },
+  onChangeLanguage: (code: string, name: string) => {}
```

Comment:
I like this passing of event handlers for configuration, in this case, I guess it would need to trigger a page reload after doing the work at some endpoint?

Reply:
yeah that'll throw up a modal to explain what's going to happen, then that POSTs to a django view, calls to mailroom to do the change, then refreshes the editor with new definition